index.html

```diff
@@ -629,7 +629,101 @@ <h3>Presentations</h3>
 that are often, but not required to be, related.
         </p>
 
-      </section>
+    </section>
+    <section>
+      <h3>Lifecycle</h3>
+
+      <em>This section is non-normative.</em>
+
+      <p>
+This section seeks to illuminate the lifecycle of a credential. Credentials 
+allow one to make statements about a subject. Moreso, it provides a succint 
+composition mechanism for tamper-evident claims that provide cryptographic 
+guarantees on the provenance of those claims. Presentations enable subjects to 
+use one or more credentials to satisfy requests from a verifier and assert 
+proof of control of those credentials. Credentials, when in motion, form a 
+general flow. This flow provides the bedrock for the lifecycle of a 
+credential, it highlights the issuance of credentials to their verification.
+      </p>
+
+      <p>
+The lifecycle of credentials will take a common path: issuance
+of claims to a credential subject, storage of credentials into credential
+repository engineered to store verifiable credentials, and then the
+presentation to a verifer. In <a href="#ex-1-example-verifiable-credential">
+Example 1</a> Pat receives an "AlumniCredential" from "Example University" that 
+will be stored in Pat's digital wallet on his mobile device, a type of 
+credential repository.
+      </p>
+
+      <pre class="example nohighlight" title="Example Verifiable Credential">
+{
+  "@context": [
+    "https://w3.org/2018/credentials/v1",
+    "https://example.com/examples/v1"
+  ],
+  "id": "http://example.edu/credentials/1872",
+  "type": ["VerifiableCredential", "AlumniCredential"],
+  "issuer": "https://example.edu/issuers/565049",,
+  "issuanceDate": "2010-01-01T19:73:24Z",
+  "credentialSubject": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "alumniOf": "Example University"
+  },
+  "proof": {
+    "type": "RsaSignature2018",
+    "created": "2017-06-18T21:19:10Z",
+    "creator": "https://example.com/jdoe/keys/1",
+    "nonce": "c0ae1c8e-c7e7-469f-b252-86e6a0e7387e",
+    "signatureValue": "BavEll0/I1zpYw8XNi1bgVg/sCneO4Jugez8RwDg/+
+      MCRVpjOboDoe4SxxKjkCOvKiCHGDvc4krqi6Z1n0UfqzxGfmatCuFibcC1wps
+      PRdW+gGsutPTLzvueMWmFhwYmfIFpbBu95t501+rSLHIEuujM/+PXr9Cky6Ed
+      +W3JT24="
+  }
+}
+      </pre>
+
+      <p>
+In the final chapter of this lifecycle, Pat's "AlumniCredential" will be 
+presented for a discount. The verifier, a ticket sales counter, states that any
+alumni of "Example University" will receive a 72% discount on football season 
+tickets. Pat uses thier mobile device to select the Verifiable Credential 
```

Comment:
"Pat uses his mobile device . . ."

Reply:
Pat is neither a man or a woman... Pat is without gender on purpose. :)

Reply:
"thier" should be "their"